fbw-a380x/src/systems/systems-host/systems/LegacyGpws.ts

```diff
@@ -0,0 +1,791 @@
+import { Arinc429Word, NXDataStore } from '@flybywiresim/fbw-sdk';
+import { FmgcFlightPhase } from '@shared/flightphase';
+import { LegacySoundManager, soundList } from 'systems-host/systems/LegacySoundManager';
+import { RadioAutoCallOutFlags } from '../../../../../fbw-a32nx/src/systems/shared/src/AutoCallOuts';
+
+/** The default (Airbus basic configuration) radio altitude auto call outs. */
```

Comment:
```suggestion
// FIXME these are the A320 defaults. Need to confirm A380.
/** The default (Airbus basic configuration) radio altitude auto call outs. */
```

Reply:
We confirmed this config for Emirates A380, this seems to be the config used in them.